libqtile/widget/base.py

```diff
@@ -170,6 +170,7 @@ def __init__(self, length, **config):
 
         self.configured = False
         self._futures: list[asyncio.TimerHandle] = []
+        self.mirrors: set[_Widget] = set()
```

Comment:
It looks like `self.mirrors` is only accessed by `self` - if that's the case could we make it `_mirrors`?

Reply:
Sure.